backend/src/routes/api/modelRegistries/modelRegistryUtils.ts

```diff
@@ -0,0 +1,300 @@
+import { MODEL_REGISTRY_NAMESPACE } from '../../../utils/constants';
+import { KubeFastifyInstance, ModelRegistryKind, RecursivePartial } from '../../../types';
+import { PatchUtils, V1Secret, V1Status } from '@kubernetes/client-node';
+
+const MODEL_REGISTRY_API_GROUP = 'modelregistry.opendatahub.io';
+const MODEL_REGISTRY_API_VERSION = 'v1alpha1';
+const MODEL_REGISTRY_PLURAL = 'modelregistries';
+
+const base64encode = (value?: string): string => {
+  // This usage of toString is fine for encoding
+  // eslint-disable-next-line no-restricted-properties
+  return Buffer.from(value || '').toString('base64');
+};
+const base64decode = (encodedValue?: string): string =>
```

Comment:
Mmmm yeah, great idea having the utility function, we've always encoded/decoded without having a declared function in the backend